translate/src/modules/otherlocales/components/Translation.tsx

```diff
@@ -46,9 +47,13 @@ export function Translation({
     if (window.getSelection()?.isCollapsed !== false) {
       setElement(index);
       setCopied(true);
-      setEditorFromMachinery(translation.translation, [], true);
+      const value =
+        format === 'ftl'
+          ? getSimplePreview(translation.translation)
+          : translation.translation;
+      setEditorFromMachinery(value, [], true);
```

Comment:
`setEditorFromHelpers` would be a better name, because the Locales tab is not part of Machinery.